test/service/test_manifest.py

```diff
@@ -204,6 +207,41 @@ def test_pfb_manifest(self):
                         with open(results_file, 'w') as f:
                             json.dump(records, f, indent=4, sort_keys=True)
 
+    def shared_file_bundle(self, bundle):
+        bundle = self._load_canned_bundle(bundle)
+
+        process = bundle.metadata_files['process_0.json']
+        process_id = '17354879-5237-4f67-9954-7cdab2ee8d0a'
+        process['provenance']['document_id'] = process_id
+        bundle.manifest[1]['uuid'] = process_id
```

Comment:
Either locate the manifest entry dynamically or assert that entry 1 is the right one.

Reply:
Using string replace instead.